tools/cloud-machines/README.md

```diff
@@ -0,0 +1,9 @@
+Setting up VM on the CERN cloud
+===============================
+
+1. [Subscribe to the Cloud service](http://clouddocs.web.cern.ch/clouddocs/tutorial_using_a_browser/subscribe_to_the_cloud_service.html)
+2. Upload your public ssh key to the Cloud service [access and security](https://openstack.cern.ch/dashboard/project/access_and_security/)
+3. `source setup-lxplus.sh`
```

Comment:
This file is called [`lxplus-setup.sh`](https://github.com/betatim/first-analysis-steps/blob/vm-setup/tools/cloud-machines/lxplus-setup.sh).